docs/tutorial/images.rst

```diff
@@ -12,73 +12,88 @@ inside of it, run::
 
     import lumicks.pylake as lk
 
-    file = lk.File("example.h5")
+    file = lk.File("Scan_WTCas9_2Markers_unspecific.h5")
     list(file.scans)  # e.g. shows: "['reference', 'bleach', 'imaging']"
 
-Once again, :attr:`.scans <lumicks.pylake.File.scans>` is a regular Python dictionary so we can
-easily iterate over it::
+:attr:`.scans <lumicks.pylake.File.scans>` is a regular Python dictionary so we can iterate over it::
 
     # Plot all scans in a file
     for name, scan in file.scans.items():
         scan.plot(channel="rgb")
         plt.savefig(name)
 
-Or just pick a single one::
+Or just pick a single one by providing the name of the scan as ``scan=file.scans["name"]``::
 
-    scan = file.scans["name"]
+    scan = file.scans["41"]
     scan.plot("red")
 
-Scan data and details
----------------------
+Plotting and Exporting
+----------------------
 
-You can access the raw image data directly. For a :class:`~lumicks.pylake.scan.Scan` with only a single frame::
+As shown above, there are convenience functions for plotting either the full RGB image or a single
+color channel::
 
-    rgb = scan.get_image("rgb")  # matrix with `shape == (h, w, 3)`
-    blue = scan.get_image("blue")  # single color so `shape == (h, w)`
+    scan.plot(channel="rgb")
 
-    # Plot manually
-    plt.imshow(rgb)
+The `channel` argument accepts the strings “red”, “green”, “blue”, or “rgb”. Multi-frame scans are also supported::
```

Comment:
```suggestion
The `channel` argument accepts the strings `“red”`, `“green”`, `“blue”`, or `“rgb”`. Multi-frame scans are also supported::
```
Very minor, but might be nice to emphasize that they're literal code strings by making them a code block.